scipy/sparse/csgraph/_laplacian.py

```diff
@@ -13,7 +13,8 @@
 
 ###############################################################################
 # Graph laplacian
-def laplacian(csgraph, normed=False, return_diag=False, use_out_degree=False):
+def laplacian(csgraph, normed=False, return_diag=False, use_out_degree=False,
+              inplace=False):
```

Comment:
```suggestion
def laplacian(csgraph, normed=False, return_diag=False, use_out_degree=False,
              *, inplace=False):
```

Reply:
why? I do not understand

Reply:
This is to make it keyword only. When we add new parameters, we do this now so we are free to move them around. It's also clearer in user code.